integration_test/EmitVerilog/sv-structs.mlir

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: verilator
```

Comment:
Yep, this is exactly what I was looking for. The "verilator" requirement seems a too specific though. I'm pretty sure this'll work in any rtl simulator.

Reply:
True, I will see if we can open it up.

Reply:
Why do we need an integration test here?  It seems that the `ExportVerilog/sv-dialect.mlir` is perfectly fine and runs much faster.

Reply:
I think there was some concern about whether the SV we are now emitting is actually going to have the intended behavior. If it's sufficient to test this once, offline, and check in the unit test with the confirmed working SV output, that is fine by me. @teqdruid thoughts?

Reply:
If anyone ends up here, the discussion is in this issue now: https://github.com/llvm/circt/issues/1368